src/lib/MAVLink/MAVLink.cpp

```diff
@@ -105,3 +105,24 @@ void convert_mavlink_to_crsf_telem(uint8_t *CRSFinBuffer, uint8_t count, Handset
     }
 #endif
 }
+
+bool isThisAMavPacket(uint8_t *buffer, uint16_t bufferSize)
+{
+#if !defined(PLATFORM_STM32)
+    for (uint8_t i = 0; i < bufferSize; ++i)
+    {
+        uint8_t c = buffer[i];
+
+        mavlink_message_t msg;
+        mavlink_status_t status;
+
+        // Try parse a mavlink message
+        if (mavlink_parse_char(MAVLINK_COMM_0, c, &msg, &status))
```

Comment:
I think you need to use another set of buffers here. `convert_mavlink_to_crsf_telem` is using `MAVLINK_COMM_0` 
```suggestion
        if (mavlink_parse_char(MAVLINK_COMM_1, c, &msg, &status))
```
Will need to bump `MAVLINK_COMM_NUM_BUFFERS` once https://github.com/ExpressLRS/ExpressLRS/pull/2846 is merged.

Reply:
Will do and thanks for the review!

Reply:
I was about to make this change but it mightnt be required.

This buffer is only used before the tx module starts to transmit and before a vehicle connection.  After a connection is made it is no longer used, and can be used freely by convert_mavlink_to_crsf_telem().

Is this ok, or is there something Im missing with how the buffer is being use?

Reply:
Hum, if there never used at that same time then I guess its OK. If they do ever get used at once then messages will be interleaved resulting in corruption of both. Might be worth a comment to warn future us about that.